about.html

```diff
@@ -0,0 +1,116 @@
+                <a href="#" class="menu-item nav-item nav-link">News</a>
+                <button type="button" class="menu-button d-none d-md-block">WebDev Campaign</button>
+            </div>
+        </nav>
+        <div class="headline mt-4">
+            <div class="container about-hl-wrapper text-center">
+                <p>Sharing the knowledge</p>
+                <h1>WebDev Summit 2022</h1>
+                <div class="about-hl-box border border-2 bg-white">
+                    <p>The WebDev Summit brings together all the important figures of the web development world. It is a unique experience where knowledge is celebrated and shared.</p>
+                </div>
+                <p class="about-bottom-text">Please contact us per Email for any further questions about WebDev Summit 2022 <br><a href="mailto:webdev22@domain.com">webdev22@domain.com</a></p>
+            </div>
+        </div>
+    </header>
+    <main>
+        <section class="container webdev-logo pt-4 pb-4">
+            <h2 class="text-center">WebDev Summit 2022 Logo</h2>
+            <hr class="section-hr mx-auto">
+            <p class="text-center">The logo for the 2022 version of WebDev Summit was decided by popular choice between 10 different proposals</p>
+            <div class="logo-box">
+                <p class="display-4 fw-bold text-center">WebDev '22</p>
+            </div>
+        </section>
+        <section class="past-summits container-fluid pt-4 pb-4 border-top">
+            <h2 class="text-center">See the past WebDev Summits</h2>
+            <hr class="section-hr mx-auto mb-4">
+            <p class="text-center">Take a look at the past WebDev Summits that took place in Canda and Japan</p>
+            <div class="container past-container row mt-4 mx-auto d-md-flex justify-content-md-around">
+                <div class="summit summit-1 col-12 col-md-6 d-flex flex-column align-items-center justify-content-center">
+                    <h4>2018</h4>
+                    <p class="summit-text">WebDev Summit 2018 in Canada</p>
+                </div>
+                <div class="summit summit-2 col-12 col-md-6 d-flex flex-column align-items-center justify-content-center">
+                    <h4>2015</h4>
+                    <p class="summit-text">WebDev Summit 2015 in Japan</p>
+                </div>
+            </div>
+        </section>
+        <section class="partner container-fluid mt-4 pt-4 pb-4 d-md-none">
+            <h2 class="text-white text-center">Partner</h2>
+            <hr class="section-hr mx-auto mb-4">
+            <div class="partner-wrapper d-flex flex-wrap">
+                <div class="partner-div w-50 w-md-25">
+                    <img src="img/Mozilla_Foundation_logo_(pre_2017).svg" alt="" class="img-fluid">
+                </div>
+                <div class="partner-div w-50 w-md-25">
+                    <img src="img/Airbnb_Logo_Bélo.svg" alt="" class="img-fluid">
```

Comment:
Kindly, the Alt attributes should contain a description of your image, that will greatly help the accessibility of your website.